src/platform/user/profile/vet360/containers/AddressValidationModal.jsx

```diff
@@ -121,7 +121,7 @@ class AddressValidationModal extends React.Component {
         className={
           isFirstOptionOrEnabled
             ? 'vads-u-display--flex vads-u-flex-direction--column vads-u-justify-content--center'
-            : 'vads-u-margin-left--2 vads-u-margin-bottom--1p5 vads-u-justify-content--center vads-u-display--flex vads-u-flex-direction--column'
+            : 'vads-u-display--flex vads-u-flex-direction--column vads-u-justify-content--center vads-u-margin-left--2 vads-u-margin-bottom--1p5'
```

Comment:
just reordered these so you can easily tell the difference between the true and false branches of the ternary.